docs/source/data-access-api/index.rst

```diff
@@ -16,21 +16,44 @@ that the full implementation of the functionality is yet to be completed.*
 
 The API can be accessed from the following URL [WIP]: https://api.dev.imap-mission.com
 
+To Install
+==========
```

Comment:
```suggestion
To Install
----------
```

I think these section headers need to be changed to 'subsections'. I notice that the way these are currently defined is messing up the Table of Contents on the main landing page.